src/main/java/org/spongepowered/api/event/entity/living/player/fishing/PlayerRetractFishingLineEvent.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered.org <http://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.event.entity.living.player.fishing;
+
+import com.google.common.base.Optional;
+import org.spongepowered.api.entity.Entity;
+import org.spongepowered.api.event.ExperienceEvent;
+import org.spongepowered.api.event.entity.living.human.fishing.HumanFishEvent;
+import org.spongepowered.api.event.entity.living.human.fishing.HumanRetractFishingLineEvent;
+import org.spongepowered.api.item.inventory.ItemStack;
+
+import javax.annotation.Nullable;
+
+/**
+ * Called when a {@link org.spongepowered.api.entity.player.Player} retracts
+ * a fishing line.
+ */
+public interface PlayerRetractFishingLineEvent extends HumanRetractFishingLineEvent, PlayerFishEvent {
+
+    /**
+     * Gets the {@link org.spongepowered.api.item.inventory.ItemStack} that will be given, if available.
+     *
+     * @return The {@link org.spongepowered.api.item.inventory.ItemStack} that will be given
+     */
+    Optional<ItemStack> getCaughtItem();
```

Comment:
Why do you declare the methods twice?
They are inherited from the human version, arent they?